src/components/calcite-flow-item/calcite-flow-item.e2e.ts

```diff
@@ -21,6 +21,16 @@ describe("calcite-flow-item", () => {
     expect(singleActionContainer).toBeNull();
   });
 
+  it("should not render leading actions container when there are no leading actions", async () => {
```

Comment:
@asangma looks good. Can you just add a test that does the opposite? `"should render leading actions container when there are leading actions"`

Reply:
No problem!

Reply:
@driskull Done!

Reply:
Looks good!

Reply:
Thanks @driskull!
Merged in master. Will MURGL when Travis is done.